01-introduction.Rmd

```diff
@@ -1,6 +1,6 @@
 # Introduction {#intro}
 
-This book is about harnessing the power of modern computers to make the most of geographic data.
+This book is about harnessing the power of modern computers to 'do' exciting things with geographic data.
```

Comment:
I think 'make the most of' was marginally better but not as good as '*do things* with' although I'm confident we could find a better wording and it's probably not worth thinking too much about it at this stage. Any suggestions @Nowosad ?